src/rest-server/src/models/v2/user.js

```diff
@@ -19,59 +19,38 @@
 // module dependencies
 const crudUtil = require('@pai/utils/manager/user/crudUtil');
 const user = require('@pai/utils/manager/user/user');
-const authConfig = require('@pai/config/authn');
+const groupModel = require('@pai/models/v2/group');
+
 
 const crudType = 'k8sSecret';
 const crudUser = crudUtil.getStorageObject(crudType);
 const crudConfig = crudUser.initConfig(process.env.K8S_APISERVER_URI);
 
+// crud user wrappers
 const getUser = async (username) => {
-  try {
-    return await crudUser.read(username, crudConfig);
-  } catch (error) {
-    throw error;
-  }
+  return await crudUser.read(username, crudConfig);
 };
 
 const getAllUser = async () => {
-  try {
-    return await crudUser.readAll(crudConfig);
-  } catch (error) {
-    throw error;
-  }
+  return await crudUser.readAll(crudConfig);
 };
 
 const createUser = async (username, value) => {
-  try {
-    return await crudUser.create(username, value, crudConfig);
-  } catch (error) {
-    throw error;
-  }
+  return await crudUser.create(username, value, crudConfig);
 };
 
 const updateUser = async (username, value, updatePassword = false) => {
-  try {
-    return await crudUser.update(username, value, crudConfig, updatePassword);
-  } catch (error) {
-    throw error;
-  }
+  return await crudUser.update(username, value, crudConfig, updatePassword);
 };
 
 const deleteUser = async (username) => {
-  try {
-    return await crudUser.remove(username, crudConfig);
-  } catch (error) {
-    throw error;
-  }
+  return await crudUser.remove(username, crudConfig);
 };
 
+// it's an inplace encrypt!
 const getEncryptPassword = async (userValue) => {
-  try {
     await user.encryptUserPassword(userValue);
     return userValue;
-  } catch (error) {
```

Comment:
This time is fine. But next time, I think the unrelative change should be in another PR.